package.json

```diff
@@ -8,7 +8,8 @@
     "test": "karma start",
     "prepublish": "typings install && webpack && webpack --config webpack.vendor.config.js",
     "build-docs": "jsdoc -c jsdoc-conf.json",
-    "bower": "bower install"
+    "bower": "bower install",
+    "postinstall": "bower install"
```

Comment:
Hey, I must have missed this over the Holidays. This is a good catch @himdel!
Changing to `preinstall`.